core/src/test/resources/application-it.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2018 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+grpc.server.port=${GRPC_PORT:6666}
```

Comment:
By the way, do we need to set these env var parameters? If we set `GRPC_SERVER_PORT` then it will replace this variable.